src/main/resources/messages.properties

```diff
@@ -287,16 +287,16 @@ screen.surrogates.button.cancel=Cancel
 screen.surrogates.account.selection.error=You are not authorized to impersonate the indicated user at this time.
 
 # Password policy
-password.expiration.warning=Your password expires in {0} day(s). Please <a href="https://pm.example.edu">change your password</a> now.
-password.expiration.loginsRemaining=You have {0} login(s) remaining before you <strong>MUST</strong> change your password.
+password.expiration.warning=Your password expires in {0} day(s). Please <a href="https://georchestra-127-0-1-1.traefik.me/console/account/passwordRecovery">change your password</a> now.
+password.expiration.loginsRemaining=You have {0} login(s) remaining before you <strong>MUST</strong> <a "href=https://georchestra-127-0-1-1.traefik.me/console/account/passwordRecovery">change your password</a>.
 screen.accountdisabled.heading=This account has been disabled.
 screen.accountdisabled.message=Please contact the system administrator to regain access.
 screen.accountlocked.heading=This account has been locked.
 screen.accountlocked.message=Please contact the system administrator to regain access.
 screen.expiredpass.heading=Your password has expired.
-screen.expiredpass.message=Please <a href="{0}">change your password</a>.
+screen.expiredpass.message=Please <a href="https://georchestra-127-0-1-1.traefik.me/console/account/passwordRecovery">change your password</a>.
 screen.mustchangepass.heading=You must change your password.
-screen.mustchangepass.message=Please <a href="{0}">change your password</a>.
+screen.mustchangepass.message=Please <a "href=https://georchestra-127-0-1-1.traefik.me/console/account/passwordRecovery">change your password</a>.
```

Comment:
The URLs will only be valid in the context of the docker composition. How about using host-relative urls instead ?
("https://..." -> "/console/...").

Reply:
Also if it was templated (`{0}`) before your modification on line 297 & 299, maybe CAS already has a way to provide the expected url ?

Reply:
@pmauduit , In fact Urls was defined inside dependency cas-server-support-thymeleaf-6.3.7.4.jar which is extracted at gradle build process to generate cas templates

Reply:
> The URLs will only be valid in the context of the docker composition. How about using host-relative urls instead ? ("https://..." -> "/console/...").

I think URL should be configurable, at build process or something like that